node/src/accountant/test_utils.rs

```diff
@@ -889,8 +889,10 @@ impl PendingPayableDaoFactoryMock {
     }
 }
 
-impl Default for PayableScanner {
-    fn default() -> Self {
+pub type PayableScannerBuilder = PayableScanner;
```

Comment:
I truly think it's safe to use, since it's a builder type, it doesn't matter whether we use default or some function. I think you're confusing that Default must always be used in production and never in test.